torchao/quantization/GPTQ.py

```diff
@@ -893,6 +893,34 @@ def _replace_linear_8da4w(
     linear_class: Type[torch.nn.Module],
     copy_weights: bool = False,
 ):
+
+    from torchao.quantization.quant_api import _replace_with_custom_fn_if_matches_filter
```

Comment:
This should be moved to the top of the file

Reply:
added a comment here "import the util function here to avoid circular dependency"